src/util/logger.js

```diff
@@ -0,0 +1,22 @@
+import pino from 'pino';
+
+
+const isDev = process.env.NODE_ENV !== 'production';
+const isTest = process.env.NODE_ENV === 'test';
+
+const reqSerializerDev = req => `${req.method.toUpperCase()} ${req.url.href}`;
+const resSerializerDev = res => res.statusCode;
+
+const logger = pino({
+	prettyPrint: isDev,
+	timestamp: isDev,
+	messageKey: 'message',
+	enabled: !isTest,
```

Comment:
I'm assuming this means that during the tests, if  you run jest with `-- verbose` you won't see any of the logging. If that's the case, sometimes I do use logging in methods and tests to debug a failing test or while writing new ones. Not sure if anyone else does, but i might be nice to keep it enabled all the time since we default run jest in `-- silent`